src/main/java/de/szut/lf8_project/lf8/project/dto/ProjectGetDto.java

```diff
@@ -0,0 +1,24 @@
+package de.szut.lf8_project.lf8.project.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class ProjectGetDto {
+    private Long id;
+    private Long customerId;
+    private Long responsibleEmployeeId;
+    private Long responsibleCustomerEmployeeId;
+    private String description;
+    private List<Long> employees;
+    private List<String> skills;
```

Comment:
Wir sollten Konsistent bleiben und überall SkillSet verwenden

Reply:
lets not forget our variable naming rules, if we are renaming this variable to SkillSet, It should be "skillSet" and not "SkillSet"